data/hai/hai ships.txt

```diff
@@ -699,6 +701,112 @@ ship "Ladybug" "Ladybug (Tracker)"
 	turret "Pulse Turret"
 
 
+
+ship "Modified Shield Beetle"
+	sprite "ship/hai modified shield beetle"
+	thumbnail "thumbnail/hai modified shield beetle"
+	attributes
+		category "Heavy Warship"
+		"cost" 25400000
+		"shields" 40000
+		"hull" 10000
+		"required crew" 59
+		"bunks" 78
+		"mass" 720
+		"drag" 9.1
+		"heat dissipation" .35
+		"fuel capacity" 600
+		"cargo space" 80
+		"outfit space" 855
+		"weapon capacity" 336
+		"engine capacity" 145
+		# any ship with this can be remotely blown by the Alphas or Unfettered
+		"unfettered remote" 1
+		weapon
+			"blast radius" 500
+			"shield damage" 5500
+			"hull damage" 3000
+			"hit force" 3600
+	outfits
+		"Ionic Blaster Prototype" 4
+		"Hai Tracker Pod" 4
+		"Tracker Storage Pod" 2
+		"Ionic Turret Prototype" 2
+		"Warder Anti-Missile"
+		"Value Detector"
+		"Hai Cuttlefish Jammer" 2
+		"Boulder Reactor" 2
+		"Hai Ravine Batteries"
+		"Hai Diamond Regenerator" 3
+		"Hai Zephyr Cooling" 3
+		"Hai Tracker" 252
+		"Outfits Expansion" 4
+		`"Bondir" Atomic Thruster`
+		`"Bufaer" Atomic Steering`
+		"Hyperdrive"
+		"Quantum Keystone" 2
+	engine -45 123
+	engine 45 123
+	gun -22 -103
+	gun 22 -103
+	gun -34.5 -68
+	gun 34.5 -68
+	gun -48 -53.5 "Hai Tracker Pod"
+	gun 48 -53.5 "Hai Tracker Pod"
+	gun -61.5 -23 "Hai Tracker Pod"
+	gun 61.5 -23 "Hai Tracker Pod"
+	turret 0 5 "Warder Anti-Missile"
+	turret -55.5 74.5
+	turret 55.5 74.5
+	leak "leak" 35 45
+	leak "flame" 60 95
+	leak "big leak" 50 45
+	explode "tiny explosion" 60
+	explode "small explosion" 51
+	explode "medium explosion" 54
+	explode "large explosion" 53
+	explode "huge explosion" 22
+	"final explode" "final explosion large"
+	description "This is a prototype created by the Unfettered. They have upgraded its shield matrix as well as most other components of the ship, but this process has sacrificed stability and increased heat generation."
+
+# Si-Ra will use this modified beetle when provoked,
+# he risked everything to upgrade his ship and to kill you so that he can get back into Midas
+ship "Modified Shield Beetle" "Si-Ra' angry Beetle"
```

Comment:
```suggestion
ship "Modified Shield Beetle" "Si-Ra's angry Beetle"
```

Reply:
~~even the ship names? And the comments too ig~~
Is nowhere safe? x)